hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/CustomMessageCallProcessor.java

```diff
@@ -265,7 +265,7 @@ private void doTransferValueOrHalt(
     private void doHaltIfInvalidSystemCall(
```

Comment:
This method name seems too generic.  It's only called in one place on the specific condition that the address is invalid, and it only returns one specific status code.  Shouldn't the name reflect that?